scripts/in_container/run_test_package_import_all_classes.sh

```diff
@@ -18,13 +18,23 @@
 # shellcheck source=scripts/in_container/_in_container_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_in_container_script_init.sh"
 
+setup_backport_packages
+
 echo
 echo "Testing if all classes in import packages can be imported"
 echo
 
 OUT_FILE_PRINTED_ON_ERROR=$(mktemp)
 
-if [[ ! ${INSTALL_AIRFLOW_VERSION:=""} =~ 1.10* ]]; then
+if [[ ${INSTALL_AIRFLOW_VERSION:=""} == "2.0.0-dev"  ]]; then
```

Comment:
same here 
```suggestion
if [[ ${INSTALL_AIRFLOW_VERSION:=""} =~ "2.0.0-*"  ]]; then
```

WDYT?

Reply:
Ditto =~ 2.*